yarn.lock

```diff
@@ -6549,56 +6552,70 @@ error-ex@^1.3.1:
   dependencies:
     is-arrayish "^0.2.1"
 
-es-abstract@^1.22.1:
-  version "1.22.3"
-  resolved "https://registry.yarnpkg.com/es-abstract/-/es-abstract-1.22.3.tgz#48e79f5573198de6dee3589195727f4f74bc4f32"
-  integrity sha512-eiiY8HQeYfYH2Con2berK+To6GrK2RxbPawDkGq4UiCQQfZHb6wX9qQqkbpPqaxQFcl8d9QzZqo0tGE0VcrdwA==
-  dependencies:
-    array-buffer-byte-length "^1.0.0"
-    arraybuffer.prototype.slice "^1.0.2"
-    available-typed-arrays "^1.0.5"
-    call-bind "^1.0.5"
-    es-set-tostringtag "^2.0.1"
+es-abstract@^1.22.1, es-abstract@^1.22.3:
```

Comment:
Nitpick: Are all these additional bumps related to the monorepo bump? 🤔 
At first sight, it seems like a "job" for "lock file maintenance". 🙈 🤷

Reply:
I just run `yarn upgrade @mui/monorepo` and `yarn deduplicate`

Reply:
🙈 
Have you tried only updating what's necessary?
Try deleting the `monorepo` entry in the lock file and running `yarn`. 🤷

Reply:
Seems to work 🙈🤯

Reply:
Nice. 👏